Libraries/LibGfx/TTFont.h

```diff
@@ -232,6 +263,24 @@ class Font {
             static Glyph simple(const ByteBuffer& slice, u16 num_contours, i16 xmin, i16 ymin, i16 xmax, i16 ymax);
             static Glyph composite(const ByteBuffer& slice); // FIXME: This is currently just a dummy. Need to add support for composite glyphs.
             AABitmap raster(float x_scale, float y_scale) const;
+            int ascender() const
+            {
+                if (m_type == Type::Simple) {
+                    return m_meta.simple.ymax;
+                } else {
```

Comment:
No need for `else` after a `return` statement. Same comment repeated throughout.